### ISSUE TITLE
DateTimePicker does not respect isMonthPickerVisible prop

### ISSUE THREAD

**Issue by robert-lindstrom** [SharePoint/sp-dev-fx-controls-react#283]
#### Category
[ ] Enhancement

[x ] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: 1.13.0

#### Expected / Desired Behavior / Question
Setting the isMonthPickerVisible prop to false should make the month picker not appear.

#### Observed Behavior
Month picker always appears irrespective of the value of the isMonthPickerVisible prop.

#### Steps to Reproduce
Setting isMonthPickerVisible to false as below has no effect
```typescript
<DateTimePicker
  label="MyLabel"
  isMonthPickerVisible={false}
  onChange={(value) => console.log("DateTimePicker value:", value)}
/>
```

**Comment by estruyf** [comment]
Will be available in version `1.8.2` and can already be tested out in the latest beta. More information about how you can test out a beta version can be found here: https://sharepoint.github.io/sp-dev-fx-controls-react/beta/

**Comment by estruyf** [comment]
This has just been released in the version `1.13.2`.
